eth2util/deposit/deposit.go

```diff
@@ -30,34 +31,27 @@ import (
 
 var (
 	// https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/validator.md#eth1_address_withdrawal_prefix
-	eth1AddressWithdrawalPrefix = byte(0x01)
+	eth1AddressWithdrawalPrefix = []byte{0x01}
 
 	// the amount of ether in gwei required to activate a validator.
 	validatorAmt = eth2p0.Gwei(32000000000)
 
-	// zeroes11 refers to a zeroed out 11 byte array.
-	zeroBytes11 = []byte{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00}
-
-	// zeroes refers to a zeroed out 32 byte array.
-	zeroBytes32 = []byte{0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00}
-
 	// DOMAIN_DEPOSIT. See spec: https://benjaminion.xyz/eth2-annotated-spec/phase0/beacon-chain/#domain-types
 	depositDomainType = eth2p0.DomainType([4]byte{0x03, 0x00, 0x00, 0x00})
 )
 
-// GetMessageRoot returns the hash root of the deposit message.
-func GetMessageRoot(pubkey eth2p0.BLSPubKey, withdrawalAddr string) (eth2p0.Root, error) {
+// getMessageRoot returns a deposit message hash root created by the parameters.
+func getMessageRoot(pubkey eth2p0.BLSPubKey, withdrawalAddr string) (eth2p0.Root, error) {
 	creds, err := withdrawalCredsFromAddr(withdrawalAddr)
 	if err != nil {
-		return eth2p0.Root{}, errors.Wrap(err, "withdrawal credentials")
```

Comment:
only wrap external packages, no need to wrap Charon packages or Charon functions (since the errors have proper stack traces)

Reply:
noted!